app/assets/stylesheets/application.css

```diff
@@ -0,0 +1,84 @@
+/*
+ * This is a manifest file that'll be compiled into application.css, which will include all the files
+ * listed below.
+ *
+ * Any CSS and SCSS file within this directory, lib/assets/stylesheets, vendor/assets/stylesheets,
+ * or any plugin's vendor/assets/stylesheets directory can be referenced here using a relative path.
+ *
+ * You're free to add application-wide styles to this file and they'll appear at the bottom of the
+ * compiled file so the styles you add here take precedence over styles defined in any styles
+ * defined in the other CSS/SCSS files in this directory. It is generally better to create a new
+ * file per style scope.
+ *
+ *= require_tree .
+ *= require_self
+ */
+
+ * {
+   box-sizing: border-box;
+ }
+
+ html {
+   background-image: url('../assets/corkboard.jpg');
+ }
+
+ .container {
+   background-color: white;
+   padding-left: 2em;
+   padding-right: 2em;
+   padding-bottom: 2em;
+   margin: 4em;
+   box-shadow: 10px 10px 5px rgba(0, 0, 0, .5);
+ }
+
+ .tack {
+   height: 50px;
+   display: block;
+   margin-left: auto;
+   margin-right: auto;
+ }
+
+
+ table {
+   border-collapse: collapse;
+   margin-bottom:10px;
+ }
+
+ th, td {
+   border-bottom: 1px solid #cecfd5;
+   padding: 10px 15px;
+ }
+
+ .call_to_action {
+   margin-top: 40px;
+ }
+
+ /******** FORM FORMATTING *********/
```

Comment:
Hey, thank you. Good job comments are a really helpful (and much needed) morale booster. I appreciate it. Organization rulez.